services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeError.kt

```diff
@@ -29,7 +29,7 @@ enum class HankeError(val errorMessage: String) {
     HAI1032("Invalid Hankealue data"),
     HAI2001("Application not found"),
     HAI2002("Incompatible application data type"),
-    HAI2003("Application is already processing in Allu and can no longer be updated."),
+    HAI2003("Application situation in Allu prevents operation."),
```

Comment:
I think this is too vague. It might be better to have a different exception and error code for the new situation.

Reply:
This is now separated to different exception and HAI2003 remains untouched.